src/Main.java

```diff
@@ -0,0 +1,87 @@
+import java.util.ArrayList;
+import java.util.Random;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Pikachu pikachu =new Pikachu("Ash");
+        Charmander charmander =new Charmander("Ash");
+        Squirtle squirtle =new Squirtle("Ash");
+        Bulbasaur bulbasaur =new Bulbasaur("Ash");
+
+        Pokemon[] pokemons = {pikachu,charmander,squirtle,bulbasaur};
+
+        // pet them all
+        for( int i=0; i<pokemons.length; i++){
+            pokemons[i].pet();
+        }
+
+
+        // do the base attacks
+        System.out.println("\n\nWarmup time!");
+        pikachu.baseAttack(squirtle);
+        charmander.baseAttack(bulbasaur);
+        squirtle.baseAttack(charmander);
+        bulbasaur.baseAttack(pikachu);
+
+
+        for( int i=0;i<10; i++) {
```

Comment:
Waarom staat dit in een for-loop? Wil je 10 keer een random battle royale uitvoeren?